docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -0,0 +1,91 @@
+Dockstore CLI FAQ
+=================
+
+For general FAQs not related to the Dockstore CLI, please see :doc:`our main FAQ page </faq>`.
+
+.. contents:: Table of Contents
+  :local:
+
+How does launching with Dockstore CLI compare with cwltool?
+-----------------------------------------------------------
+
+Under the hood, the Dockstore CLI evokes cwltool to launch CWL workflows. However, it adds additional features. Dockstore CLI has utilities to generate JSON parameter files from
+entries on Dockstore (``dockstore tool convert``).
+
+When launching tools, the Dockstore CLI makes it easy to specify entries
+from the Dockstore website. We can also provision input and output files using HTTP,
+FTP, S3, and GCS. As of Release 1.12, the Dockstore CLI has support for running on `a WES server <https://github.com/ga4gh/workflow-execution-service-schemas>`__. We also have preliminary support for `Synapse <https://www.synapse.org/>`__ and the `ICGC Storage
+client <https://docs.icgc.org/download/guide/#score-client-usage>`__. Please see `file provisioning plugins <https://github.com/dockstore/dockstore-cli/tree/master/dockstore-file-plugin-parent>`__
```

Comment:
It's on the next line. RST does not care about a single newline in between a sentence. 
<img width="762" alt="Screen Shot 2022-03-07 at 12 49 17 PM" src="https://user-images.githubusercontent.com/27784612/157115599-4edd9a4d-0ede-4481-a2a2-8d3577ebe3be.png">

In fact, these docs have a tendency to have a newline about once every ~66-72 characters. I think someone was using an 80 character line as their guideline. To avoid filling the changes with a bunch of deleted newlines, I tried to stick to that when editing existing docs, but I did not have the patience to stick to it when I was writing up brand new sections.

Personally I am of the belief that sticking to the 80 line convention makes more sense for code rather than docs, because docs are expected to be a bit more WYSIWYG, and sudden newlines can lead to confusion like this for reviewers.